test/_init_tests.js

```diff
@@ -79,7 +79,12 @@ global.AMP.extension = function (name, version, installer) {
 };
 
 // Make amp section in karma config readable by tests.
-window.ampTestRuntimeConfig = parent.karma ? parent.karma.config.amp : {};
+if (parent.karma && !parent.__karma__) {
+  parent.__karma__ = parent.karma;
+}
+window.ampTestRuntimeConfig = parent.__karma__
+  ? parent.__karma__.config.amp
+  : {};
```

Comment:
This was super fun to find. When I used the `debug.html` window in Karma, I suddenly got a bunch of "Timeout of NaNms exceeded" errors and none of the tests ran. It's because `karma` object exists only in the test runner mode, and `__karam__` exists only in `debug.html`.

Because of this, `ampTestRuntimeConfig.mochaTimeout` was `undefined`, and [`Math.max`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-bind/0.1/test/integration/test-bind-impl.js#L205) of `undefined` is `NaN`, and `mocha.timeout(NaN)` breaks everything.

/cc @rsimha

Reply:
Could this also be the root cause of https://github.com/ampproject/amphtml/issues/32091 reported by @samouri?

Reply:
Unrelated, see https://github.com/ampproject/amphtml/pull/32886.